File: Leetcode-Problems/Find Missing and Repeated Values.cpp
class Solution
{
public:
    vector<int> findMissingAndRepeatedValues(vector<vector<int>> &grid)
    {
        int n = grid.size();
        vector<int> arr(n * n + 1, 0);
        for (int i = 0; i < grid.size(); i++)
        {
            for (int j = 0; j < grid[i].size(); j++)
            {
                int temp = grid[i][j];
                arr[temp] += 1;
            }
        }
        int a = 0, b = 0;
        for (int i = 1; i < arr.size(); i++)
        {
            if (arr[i] == 0)
            {
                b = i;
            }
            else if (arr[i] == 2)
            {
                a = i;
            }
        }
        return {a, b};
    }
};

File: Leetcode-Problems/Find Missing and Repeated Values.java
class Solution {
    public int[] findMissingAndRepeatedValues(int[][] grid) {
        int arr[] = new int[(grid.length * grid.length) + 1];
        for (int i = 0; i < grid.length; i++) {
            for (int j = 0; j < grid.length; j++) {
                arr[grid[i][j]]++;
            }
        }
        int[] ans = new int[2];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] == 2)
                ans[0] = i;
            if (arr[i] == 0)
                ans[1] = i;
        }
        return ans;
    }
}